.pylintrc

```diff
@@ -14,7 +14,7 @@ ignore=CVS
 
 # Add files or directories matching the regex patterns to the blacklist. The
 # regex matches against base names, not paths.
-ignore-patterns=
+ignore-patterns=_scm_version\.py
```

Comment:
I cannot reproduce any problems with this :exploding_head:

Reply:
Oh, I see now. It's only reproducible with your other changes in `tox.ini` but since I'll go for a pre-commit based solution, it won't be a problem.